ui/component/youtubeWelcome/view.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
```

Comment:
Not necessary to do this now, but it looks like it'd be pretty easy to make this a generic referrer component, where a domain maps to a title, message, and button label. @finer9 something to keep in mind for the future.